.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: prepare
       run: |
+        sudo apt-get install -y doxygen
```

Comment:
Also needs graphviz to generate png images. We should also add clang-tidy.

Reply:
Good catch!